Lib/lib2to3/pgen2/tokenize.py

```diff
@@ -56,7 +56,7 @@ def _combinations(*l):
 Whitespace = r'[ \f\t]*'
 Comment = r'#[^\r\n]*'
 Ignore = Whitespace + any(r'\\\r?\n' + Whitespace) + maybe(Comment)
-Name = r'[a-zA-Z_]\w*'
+Name = r'\w(?<!\d)\w*'
```

Comment:
`Lib/tokenize.py` appears to parse numbers before identifiers to avoid having a look-behind assertion here. Can we take that approach here, too?

Reply:
@benjaminp copied codes in `Lib/tokenize.py`.

Reply:
In `Lib/tokenize.py`, I see:
```
Name = r'\w+'
```